content/getting-started/multiDeviceMultiSig/_index.en.md

```diff
@@ -0,0 +1,175 @@
+{{</ notice >}}
+
+{{< tabs >}}
+
+  {{< tab "Desktop" >}}
+  
+1. Open the chat feature located on the left side bar. <br/><br/>
+    
+<img src = desktopChatButton.png
+    alt = "Chat Button (Screenshot)"/>
+
+2. Click the "Start new message" button and type in your user email. You should see your name and email pop up below, click it!<br/><br/>
+
+<img src = desktopStartNewMessage.png
+    alt = "start new message button (Screenshot)"/>
+<img src = desktopNameSelected.png
+    alt = "Name selected (Screenshot)"/>
+
+3. Send a friendly message to yourself to initiate the chat room. After sending the message, you should see a new chat room with yourself!<br/><br/>
+
+<img src = desktopJoinedChatRoom.png
+    alt = "Wallet + button (Screenshot)"/>
+
+4. Click the "Create a collaborative wallet" button to initiate the multisig configuration setup.<br/>
+
+<img src = desktopCreateCollabWalletBttn.png
+    alt = "collaborative wallet question (Screenshot)"/>
+
+5. Name your wallet and click "Continue".<br/><br/>
+    
+<img src = desktopWalletName.png
+    alt = "Create collaborative wallet view (Screenshot)"/>
+
+6. Add the number of keys you will be using in your wallet. Then, add the required keys to sign a transaction.
+
+{{< notice tip >}}
+NOTE: The total number of keys refers to how many keys are valid for this wallet. The required keys refers to how many keys are needed to sign and send a transaction.
+{{< /notice >}}
+
+<img src = desktopWalletConfig.png
+    alt = "Configure collaborative wallet (Screenshot)">
+
+
+7. Be sure to check your configuration before clicking "Continue".<br/><br/>
+
+<img src = desktopCurrentConfig.png
+    alt = "your current configuration (Screenshot)">
+
+8. You can now review other parameters of your wallet configuration, then continue.<br/><br/>
+
+<img src = desktopReviewWallet.png
+    alt = "Review collaborative wallet (Screenshot)">
+
+
+9. Ensure your local key is selected (for me, that is my Desktop Key) and click "Continue".
+
+{{< notice tip >}}
+The key in green (Software) is an eligible key to sign with for your current device. Foreign keys are not eligible for signing locally and will need to be signed on the device they were imported or created on.
+{{< /notice >}}
+
+<img src = desktopAssignKeys.png
+    alt = "Assign key (Screenshot)">
+
+<img src = desktopSelectKey.png
+    alt = "Assign Key Desktop (Screenshot)">
+
+10. You should now see the original chat screen you created in step 2. This is where you can find the pending wallet configuration when you begin adding your keys on the next device.<br/><br/>
+
+<img src = desktopConfigDone.png
+    alt = "Chat screen pending keys (Screenshot)">
+
+{{< notice note >}}
+You do not need to continue to the other tabs if you were able to add all the keys to your wallet (number 9). If you want to add a key created from another device, continue to the next tab.
```

Comment:
There are two ways to assign keys to a wallet in a chat (step 9). You could assign them from a single device, provided that all of your keys have been synced to that device. Or you could assign keys one-by-one from the respective devices that the keys were created on. If you choose to do the latter, continue onto the next tabs.